test/Audit.IntegrationTest/PostgreSqlTests.cs

```diff
@@ -28,22 +96,15 @@ public void Test_PostgreDataProvider_CustomDataColumn()
                         auditEvent.EventType = overrideEventType;
                         return auditEvent.ToJson();
                     })
-                    .LastUpdatedColumnName("updated_date")
+                    .LastUpdatedColumnName(GetLastUpdatedColumnNameColumnName())
                     .CustomColumn("event_type", ev => ev.EventType)
                     .CustomColumn("some_date", ev => DateTime.UtcNow)
                     .CustomColumn("some_null", ev => null))
                 .WithCreationPolicy(EventCreationPolicy.InsertOnEnd)
                 .ResetActions();
-
-            var scope = AuditScope.Create("test", null);
-            scope.Dispose();
-
-            var id = scope.EventId;
+            
             var dp = (PostgreSqlDataProvider)Configuration.DataProvider;
-
-            var getEvent = dp?.GetEvent<AuditEvent>(id);
-
```

Comment:
These later lines are not actually removed.  See new lines 21-31.

I simply encapsulated GetConfiguredPostgreSqlDataProvider to avoid copy/paste code.